### DISCUSSION TITLE
how to use

### DISCUSSION THREAD

**Discussion by Welsh-Boogie** [ahmadfaizalbh/Chatbot#57] [General]
hi, im unable to understand how can I run a demo, not the demo() but the actual thing. i cant seem to get integration working. any working simple sample would be gr8. thanks

**Comment by ahmadfaizalbh** [comment]
Checkout [Example](https://github.com/ahmadfaizalbh/Chatbot/blob/master/examples/Example.py) 

If you want a web version check below
1. [HTTPServer and SQLAlcamy based](https://github.com/ahmadfaizalbh/ChatBotAI-HTTPServer)
2. [Django based](https://github.com/ahmadfaizalbh/WebBot)

**Reply by Welsh-Boogie** [reply]
hi the Example has an error for me, the Example.template is missing, how do I get that (sorry I'm new to this and just curious)


HTTPServer and SQLAlcamy based                                       site is not found

**Reply by ahmadfaizalbh** [reply]
Example.template : https://github.com/ahmadfaizalbh/Chatbot/blob/master/examples/Example.template

HTTPServer and SQLAlcamy based : https://github.com/ahmadfaizalbh/ChatBotAI-HTTPServer

**Reply by Welsh-Boogie** [reply]
Thank you! illl look into it, the Cakebot and its API is interesting.....

**Reply by Welsh-Boogie** [reply]
https://github.com/lukalabs/cakechat

is the best chatbot i have found,,,,, it is just amazing

**Reply by padmalcom** [reply]
There ist no intent recognition in cakechat so ist is nice to play but it lacks nlu features.

**Reply by Welsh-Boogie** [reply]
I used their compiled app version, it worked pretty well, any other open source bots u know? id low to check them out
@padmalcom

**Reply by Welsh-Boogie** [reply]
@ahmadfaizalbh what are some good examples of open-source chatbots? https://github.com/lukalabs/cakechat is a good one but it doesn't have intent recognition? i have seen some with intent recognition but with only intermediate sentences

**Reply by Welsh-Boogie** [reply]
It may not have intent recognition but does provide chat related to context.
